app/controllers/album.js

```diff
@@ -0,0 +1,30 @@
+const albumsService = require('../services/albums'),
+  logger = require('../logger'),
+  message = require('../constants');
+
+exports.getAlbumsInventory = (req, res, next) => {
+  logger.info(`${message.PREVIOUS_MESSAGE} to list of albums`);
+  albumsService
+    .getAlbums()
+    .then(response => {
+      logger.info(message.MESSAGE_OK);
+      res.status(200).send(response);
+    })
+    .catch(next);
+};
+
+exports.getPhotos = (req, res, next) => {
+  const idAlbum = req.params.id;
+  if (idAlbum > 100 || isNaN(idAlbum)) {
+    throw Error('Make sure your request is correct');
```

Comment:
Maybe throw a errors bad request (status code: 400) and more descriptive too. What is the error? What is wrong in the request?

Reply:
Call the error middleware function when there is an error. 
https://expressjs.com/en/guide/error-handling.html